src/_image_wrapper.cpp

```diff
@@ -10,41 +10,40 @@
  * */
 
 const char* image_resample__doc__ =
-"Resample input_array, blending it in-place into output_array, using an\n"
-"affine transformation.\n\n"
+R"""(Resample input_array, blending it in-place into output_array, using an affine transform.
```

Comment:
No, this is C++11 raw string literal syntax; it's `R"<something>(...)<something>"` where `<something>` is an optional string to prevent ambiguity if your string might contain `)"`. I chose additional double quotes to make it look similar to a Python multiline string, but if that is confusing a different string could be chosen.